gulpfile.js

```diff
@@ -7,8 +7,13 @@ var gulp = require('gulp'),
     outDir = tsconfig.compilerOptions.outDir || 'dist',
     testPattern = 'dist/**/spec/*.js';
 
-gulp.task('build', () => {
+gulp.task('copy:typings', () => {
```

Comment:
It's a temporary fix until we get the non-custom typings for restify. (custom-typings.d.ts not copied to /dist folder like other *.d.ts files).